app/src/main/res/values/strings.xml

```diff
@@ -48,59 +49,59 @@
+    <string name="sync_finished">Done!</string>
+    <string name="sync_error">An error has occurred. You may try again.</string>
+    <string name="sync_stop_confirm_title">
+        Are you sure you want to interrupt the synchronization?
+    </string>
     <string name="sync_internet_stop_confirm_message">The sync hasn\'t finished yet.</string>
 
     <!-- Private / People Sync -->
-    <string name="sync_people_starting">Setting up</string>
-    <string name="sync_people_waiting_first_client">Waiting for the first person\nto connect</string>
-    <string name="sync_people_syncing_some">Syncing with</string>
-    <string name="sync_people_stop_confirm_message">Some people are syncing with your device at the moment.</string>
+    <string name="sync_people_starting">Getting ready…</string>
+    <string name="sync_people_waiting_first_client">Waiting for the first user…</string>
+    <string name="sync_people_syncing_some">Users currently synchronizing:</string>
+    <string name="sync_people_stop_confirm_message">
+        Some users are still synchronizing data with you.
+    </string>
 
     <!-- Hotspot Instructions -->
-    <string name="hotspot_instructions">Wi-Fi hotspot</string>
+    <string name="hotspot_instructions">Wi-Fi Hotspot</string>
     <string name="hotspot_instructions_disabled">Wi-Fi hotspot is disabled</string>
     <string name="hotspot_instructions_disabled_text">
-      Your Wi-Fi hotspot is disabled at the moment.
-      You need to enable it in your device settings to be able to Sync with People.
+      You need to enable it so that Relaynet users can connect to this device and exchange data.
     </string>
-    <string name="hotspot_instructions_open_settings">Open Wi-Fi hotspot Settings</string>
+    <string name="hotspot_instructions_open_settings">Open Wi-Fi hotspot settings</string>
     <string name="hotspot_instructions_enabled">Wi-Fi hotspot is enabled</string>
     <string name="hotspot_instructions_enabled_text">
-      Your Wi-Fi hotspot is now enabled.\nYou can start the Sync with People.
-    </string>
-    <string name="hotspot_instructions_start_sync">Start to Sync</string>
-    <string name="hotspot_instructions_stay_enabled">
-        The Wi-Fi hotspot should stay enabled throughout the synchronization with people.
+      Keep it enabled throughout the synchronization with users to avoid disruptions.
     </string>
+    <string name="hotspot_instructions_start_sync">Start sync with users</string>
 
     <!-- Settings -->
     <string name="settings">Settings</string>
-    <string name="settings_manage_storage">Manage data storage</string>
+    <string name="settings_manage_storage">Manage storage utilization</string>
     <string name="settings_select_max_storage">
-        Select the maximum amount of data the app can use on your device.
+        Specify the maximum amount of space that Relaynet users\' data can take up on your device.
```

Comment:
I usually prefer to use the curvy apostrophe (’) because it looks better and does not need escaping ;)

Reply:
Agreed! I was just lazy but it does hurt my eyes to use the escape character. Can you please reapprove?